1208-projects/my_projects.feature

```diff
@@ -0,0 +1,43 @@
+Feature: User My Projects
+
+    @2874
+    Scenario: User opens My Projects
+        Given that a User is logged in
+        When they navigate to the My Projects Page
+        Then they see a Create New Project Button
+        And they see a search field
+        And they see a list of Active Projects
+        And they see a collapsed list of Concluded Projects
+        And the lists contains Projects where the User has the role as:
+            | Project Manager       |
+            | Local Project Manager |
+            | Participants          |
+        And each Project has:
+            | Title           |
+            | Institution     |
+            | Project Manager |
+        And each Project has an Edit button if the User is Project Manager
+        And the list can be sorted by:
+            | Title           |
+            | Institution     |
+            | Project Manager |
+        And they see pagination buttons for the Concluded Projects list
+    
+    Scenario: User opens a Project's Landing Page
+        Given User opens My Projects
+        When they click the Title 
+        Then they see the Landing Page for the Project
+
+    Scenario: User Edits a Project in Project Wizard
+        Given User with access rights moves the cursor over a project
+        When they click the Edit Button for a Project
+        Then User opens a Project in the Project Wizard
```

Comment:
Denne er fortsatt litt kryptisk med begrepet "access rights", og "moves the cursor over a Project"


```suggestion
    Scenario: Project Manager Edits a Project in the Project Wizard
        Given User opens My Projects
        Any they are Project Manager for a Project
        When they click the Edit Button for a Project
        Then they see the Project Wizard
```

Reply:
Hmmm - har eg spøkelser i hus. Denne skal eg jo både ha endra på og sletta...
Linje 32 peiker tilbake til linje 19:
And each Project has an Edit button if the User is Project Manager
Eg må justere litt meir på denne, så eg avviser ditt forslag og legger til fleire roller.
Dette vart strafferunde - etter antatt målgang.